README.md

```diff
@@ -234,6 +234,9 @@ For debugging, keep the `Release` configuration (we always generate PDBs) and tu
 
 ## Release Notes
 
+### Version 1.3.0 (TBD)
```

Comment:
Please add the release date

Reply:
Done, I set it to tomorrow. Let's do this!